stash/test/staking.test.ts

```diff
@@ -94,13 +94,13 @@ describe('[Staking]', () => {
     vi.unmock('../src/repository/StakingRepository')
   })
 
-  it.skip('should mock the apy endpoint method', async () => {
+  it('should mock the apy endpoint method', async () => {
     const expectedResponse: ResponseAPY[] = [
       {
         apy: '62.027726047981274135',
-        token: "0",
+        token: '5',
         date: '16/08/2023',
-        collatorAddress: '5ChaosDANFE2q4qH3UnCUJ5ZXHms5TWwsbrx4YXNrwQh2nfk',
+        collatorAddress: '5ChaosDANFE2q4qH3UnCUJ5ZXHms5TWwsbrx4YXNrwQh2nfk', //todo: qa to check what value we should set here
```

Comment:
alith or balthathar are the collators, so 
0xf24ff3a9cf04c71dbc94d0b566f7a27b94566cac
or 
0x3cd0a705a2dc65e5b1e1205896baa2be8a07c6e0
should be fine.